### PR TITLE
Prepare release 3.0.0

### PR DESCRIPTION
**Pull request by vovac12**

### CODE REVIEW

File: misc/full_test.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+cargo t
+cargo t --features ready-to-test
+cargo t --features ready-to-test,wip
+cargo t --features private-net
+cargo t --features private-net,ready-to-test
+cargo t --features private-net,ready-to-test,wip
+cargo t --features runtime-benchmarks
+cargo t --features runtime-benchmarks,private-net
+cargo t --features runtime-benchmarks,private-net,ready-to-test
+cargo t --features runtime-benchmarks,private-net,ready-to-test,wip
```
Comment (Alexey-N-Chernyshov) [inline misc/full_test.sh:25]:
```suggestion
cargo t --features runtime-benchmarks,private-net,ready-to-test,wip

```

File: node/Cargo.toml [hunk 2]
```diff
@@ -138,17 +138,13 @@ wip = [
     "common/wip",
     "dex-api-rpc/wip",
     "liquidity-proxy-rpc/wip",
+    "beefy-light-client",
+    "beefy-light-client-rpc",
```
Comment (wer1st) [inline node/Cargo.toml:142]: Why did you move it into `wip` if it should be released?

**Review by Alexey-N-Chernyshov — APPROVED** [review]

**Review by N1ghtStorm — APPROVED** [review]

File: node/Cargo.toml (hunk 2 above)
Comment (vovac12) [reply node/Cargo.toml:142]: Beefy is not part of Federated bridge
